admin/includes/javascript_loader.php

```diff
@@ -9,14 +9,14 @@
  */
 ?>
 <script src="https://code.jquery.com/jquery-3.7.1.min.js" integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous"></script>
-<script>window.jQuery || document.write('<script src="includes/javascript/jquery.min.js"><\/script>');</script>
+<script title="jQuery check">window.jQuery || document.write('<script src="includes/javascript/jquery.min.js"><\/script>');</script>
 
 <script src="https://code.jquery.com/ui/1.14.0/jquery-ui.min.js" integrity="sha256-Fb0zP4jE3JHqu+IBB9YktLcSjI1Zc6J2b6gTjB0LpoM=" crossorigin="anonymous"></script>
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha384-aJ21OjlMXNL5UyIl/XNwTMqvzeRMZH2w8c5cRVpzpU8Y5bApTppSuUkhZXN0VxHd" crossorigin="anonymous"></script>
 <!--<script src="includes/javascript/bootstrap.min.js"></script>-->
 
 <script src="includes/javascript/jquery-ui-i18n.min.js"></script>
-<script>
+<script title="jQuery.datepicker">
```

Comment:
This is more about initializing any applicable jQuery plugins, vs datepicker specifically.